azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Perhaps move this and the next cell further up in the notebook, just below the cell where you call the AzureVMCluster?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Had it here since it talked about scaling up the workers too. Can move it down if it makes more sense</p>